src/components/SideBar/SuggestedUsers.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  Avatar,
+  Button,
+  List,
+  ListItem,
+  ListItemAvatar,
+  ListItemText,
+  Link,
+} from "@mui/material";
+import React from "react";
+import { useNavigate } from "react-router-dom";
+
+const styles = {
+  avatar: {
+    border: "none",
+    cursor: "pointer",
+    opacity: 0.75,
+    "&:hover": {
+      opacity: 1,
+    },
+  },
+  list: {
+    width: "100%",
+    height: "100%",
+    backgroundColor: "gray.light",
+  },
+  listText: {
+    color: "black",
+  },
+  userContainer: {
+    cursor: "pointer",
+    height: "25%",
+    width: "100%",
+    display: "flex",
+    flexDirection: "row",
+    ":hover": {
+      backgroundColor: "#E9EBED",
+    },
+  },
+  followButton: {
+    boxShadow: "none",
+    "&:hover": {
+      boxShadow: "none",
+      backgroundColor: "primary.main",
+    },
+  },
+  displayName: {
+    fontWeight: "bold",
+  },
+};
+
+const usersData = [
+  {
+    id: 1,
+    displayName: "John Doe",
+    username: "john_doe",
+    avatarUrl: "https://example.com/avatar1.jpg",
+  },
+  {
+    id: 2,
+    displayName: "Jane Smith",
+    username: "jane_smith",
+  },
+  {
+    id: 3,
+    displayName: "Chad Son",
+    username: "chad_son",
+  },
+  {
+    id: 4,
+    displayName: "Michael Myers",
+    username: "michael_myers",
+  },
+];
+
+const SuggestedUsers = () => {
+  const navigate = useNavigate();
+
+  return (
+    <List sx={styles.list}>
+      {usersData.map((user) => (
+        <ListItem key={user.id} sx={styles.userContainer}>
```

Comment:
Since this isn't a button, I'd want to remove the pointer styling on the cursor since it's misleading. We could turn this into a `ListItemButton` so that users can click anywhere on the button too, but I think MUI advises against having buttons within buttons because of overlapping `onClick` functions.

Personally, I'd like it if we could make the whole thing a button but not sure if we should go against what MUI recommends. Maybe try it out and see if you run into any issues?

Reply:
I applied `ListItemButton` and for some reason this doesn't give me the warning where we have a `button` as a descendant of a `button`. So I will permanently change it to `ListItemButton`. 

But even though there was no warning, there is still the issue of overlapping `onClick` functionality so I had to include an onClick handler for the child button (follow button) that stops the clicking propagation at the child and doesn't go through the DOM to the ancestor components.

Reply:
Nice! I like it